WcaOnRails/app/assets/stylesheets/competitions.scss

```diff
@@ -226,6 +226,10 @@ $venue-map-wrapper-height: 400px;
   &.competitions-map #competitions-admin {
     display: none;
   }
+  
+  #competitions-map {
```

Comment:
There are other places where we use maps (like venue location maps under "Edit venues information" [in the schedule editor](https://staging.worldcubeassociation.org/competitions/WesternChampionship2021/schedule/edit)) and we may add more in the future.

It would be best to have a single global rule in `wca.scss` (where we currently keep such specific global styles) that would apply to all maps. Inspecting the rendered map it looks like there's always a root element with `leaflet-container` class, so if we target that with CSS, it will hopefully fix the issue globally.

Reply:
Thanks for the feedback! Sorry for the messy commits, I'm not great at rebasing yet.